Rubberduck.RibbonDispatcher/Tests.bas

```diff
@@ -0,0 +1,15 @@
+Attribute VB_Name = "Tests"
+Option Explicit
+
+Public Sub Test()
```

Comment:
Please remove module. This will confuse our users unfamiliar with unit testing.

Reply:
This is the second dead-code module for the initial PoC of the COM event handling. Both such will be removed promptly.

Reply:
Actually, long gone already. Are you reviewing an old commit?